src/query-builder/SelectQueryBuilder.ts

```diff
@@ -1296,20 +1297,35 @@ export class SelectQueryBuilder<Entity> extends QueryBuilder<Entity> {
      */
     protected createOrderByExpression() {
 
-        let orderBys = this.expressionMap.orderBys;
+        const orderBys = this.expressionMap.orderBys;
 
         // if table has a default order then apply it
         if (!Object.keys(orderBys).length && this.expressionMap.mainAlias!.hasMetadata) {
-            orderBys = this.expressionMap.mainAlias!.metadata.orderBy || {};
-        }
+            const orderBys = this.expressionMap.mainAlias!.metadata.orderBy || {};
```

Comment:
yeah I understood that. Please remove all other non-related changes from that file